### PR TITLE
Readme

### LINKED ISSUE #2: ENH: Documentation  + Latex visualisation to be improved
Some elements need to be corrected in the README file.
Also the formula for MCP are hard to visualize without proper LaTeX rendering.

### PR DESCRIPTION
**Pull request by mathurinm**
closes #2 

@tomMoral this is not automated but it's not something we should do every other day, and github does not seem to plan to add math rendering to rst.

Since the penalty is hard to parse, I think the benefits outweigh the imperfection of the solution

### CODE REVIEW

**Review by agramfort — APPROVED** [review]
thx @mathurinm

**Review by tomMoral — APPROVED** [review]
LGTM!

I agree this is good enough until we get an easier to maintain solution.

I have opened issue benchopt/benchopt#259 to track this with a potential (long term) solution.
